system/data/es/systems/core.json

```diff
@@ -0,0 +1,270 @@
+                        "type": "aspect"
+                    }
+                ]
+            },
+            {
+                "label": "Opcional",
+                "entries": [
+                    {
+                        "name": "Relación",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Relación"
+                        }
+                    },
+                    {
+                        "name": "Otro aspecto",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Otro"
+                        }
+                    }
+                ]
+            }
+        ],
+        "Consequences": [
+            {
+                "entries": [
+                    {
+                        "name": "Consecuencia leve",
+                        "type": "consequence",
+                        "data": {
+                            "label": "Leve",
+                            "icon": "2"
+                        }
+                    },
+                    {
+                        "name": "Segunda consecuencia leve (Condición de habilidad)",
+                        "type": "consequence",
+                        "data": {
+                            "label": "Leve",
+                            "icon": "2"
+                        },
+                        "flags": {
+                            "fatex": {
+                                "skillReferenceSettings": {
+                                    "conjunction": 0
+                                },
+                                "skillReferences": [
+                                    {
+                                        "skill": "Físico",
+                                        "condition": 4,
+                                        "operator": 4
+                                    },
+                                    {
+                                        "skill": "Voluntad",
+                                        "condition": 4,
+                                        "operator": 4
+                                    }
+                                ]
+                            }
+                        }
+                    },
+                    {
+                        "name": "Consecuencia moderada",
+                        "type": "consequence",
+                        "data": {
+                            "label": "Moderada",
+                            "icon": "4"
+                        }
+                    },
+                    {
+                        "name": "Consecuencia grave",
+                        "type": "consequence",
+                        "data": {
+                            "label": "Grave",
+                            "icon": "6"
+                        }
+                    }
+                ]
+            }
+        ],
+        "Skills": [
```

Comment:
The same typos (accents, periods) noted for condensed.json also apply here (which makes sense since I guess it was copypasted :) Please keep them in mind.